app/routes/events/components/EventEditor/index.tsx

```diff
@@ -125,10 +144,90 @@ function EventEditor({
           name="cover"
           component={ImageUploadField}
           uploadFile={uploadFile}
-          edit={isEditPage && ((token) => setCoverPhoto(eventId, token))}
           aspectRatio={20 / 6}
-          img={event.cover}
+          img={useImageGallery ? imageGalleryUrl : event.cover}
         />
+        <Modal
+          show={showImageGallery}
+          onHide={() => setShowImageGallery(false)}
+          contentClassName={styles.imageGallery}
+        >
+          <>
+            <h1>Bildegalleri</h1>
+            <Flex wrap alignItems="center" justifyContent="space-around">
+              {imageGallery &&
+                imageGallery.map((e) => (
+                  <div key={e.key} className={styles.imageGalleryContainer}>
+                    <ConfirmModal
+                      title="Fjern fra galleri"
+                      message={`Er du sikker på at du vil fjerne bildet fra bildegalleriet? Bildet blir ikke slettet fra databasen.`}
+                      closeOnConfirm
+                      onConfirm={() =>
+                        setSaveForUse(`${e.key}:${e.token}`, false)
+                      }
+                    >
+                      {({ openConfirmModal }) => (
+                        <button
+                          className={styles.closeButton}
+                          onClick={openConfirmModal}
+                        >
+                          <Icon name="close-circle" />
+                        </button>
+                      )}
+                    </ConfirmModal>
+
+                    <Image
+                      src={e.cover}
+                      placeholder={e.coverPlaceholder}
+                      alt={`${e.cover} bilde`}
+                      onClick={() => {
+                        change('cover', `${e.key}:${e.token}`);
+                        setShowImageGallery(false);
+                        setUseImageGallery(true);
+                        setImageGalleryUrl(e.cover);
+                      }}
+                      className={styles.imageGalleryEntry}
+                    />
+                  </div>
+                ))}
+              {imageGallery.length === 0 && (
+                <h2>
+                  Det finnes ingen bilder i bildegalleriet. Hvorfor ikke laste
+                  opp et?
+                </h2>
+              )}
+            </Flex>
+          </>
+        </Modal>
+
+        <Flex wrap alignItems="center" justifyContent="space-between">
+          <Flex alignItems="center" justifyContent="space-between">
+            <Button onClick={() => setShowImageGallery(true)}>
+              Velg bilde fra bildegalleriet
+            </Button>
+            <Tooltip content="Velg et bilde som er lastet opp og markert for bruk fra tidligere arrangementer.">
+              <Icon
+                name="information-circle-outline"
+                size={20}
+                style={{
+                  cursor: 'help',
+                  marginLeft: '0.5em',
```

Comment:
Is the left margin still needed? With the recent changes to the tooltip it should not

Reply:
It looks a little wack without actually, I'll leave it for now (?)